src/main/java/rs/teslaris/core/service/impl/commontypes/NotificationServiceImpl.java

```diff
@@ -26,12 +35,21 @@
 public class NotificationServiceImpl extends JPAServiceImpl<Notification>
     implements NotificationService {
 
+    private final MessageSource messageSource;
+
     private final NotificationRepository notificationRepository;
 
     private final NewOtherNameNotificationHandler newOtherNameNotificationHandler;
 
     private final AddedToPublicationNotificationHandler addedToPublicationNotificationHandler;
 
+    private final UserAccountIndexRepository userAccountIndexRepository;
+
+    private final EmailUtil emailUtil;
+
+    @Value("${client.address}")
+    private String clientAppAddress;
```

Comment:
This is address of front-end application, right? The name of the property is not intuitive for me.

Reply:
Yes it is, should I change it to `frontend.application.address`?

Reply:
Renamed! :D